src/deepsparse/tasks.py

```diff
@@ -93,7 +93,11 @@ class SupportedTasks:
 
     @classmethod
     def check_register_task(cls, task: str):
-        if cls.is_nlp(task):
+        if task == "custom":
```

Comment:
We should enable multiple custom pipelines for the user here rather than a hardcoded single one

Reply:
Would that be via multiple task names/aliases? Right now they would use `Pipeline.create("custom", input_schema=..., process_input_fn=..., ...)` which could support any processing functions/schemas, but only via the "custom" task tag

Reply:
I changed it to `task.startswith("custom")`. That seem sufficient for now? Do we need to capture whatever is after the `custom` for something?